inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -487,7 +487,7 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         LayerParams attrs = {node->get_friendly_name(), "Activation",
                               details::convertPrecision(node->get_output_element_type(0))};
         auto res = std::make_shared<InferenceEngine::CNNLayer>(attrs);
-        res->params["type"] = "not";
+        res->type = "not";
```

Comment:
It is not correct because if I remember right in IR v7 we didn't have layer with type "not". "not" type should be a layer attribute.

Reply:
Will investigate that tomorrow, but the previous version was failing, and this one was passing the test, but the build is currently failing on windows because of that operator